public/vendor/nova/mix-manifest.json

```diff
@@ -1,8 +1,8 @@
 {
-    "/app.js": "/app.js?id=170a9af8be71e40405b81537a527b2a7",
-    "/manifest.js": "/manifest.js?id=d75058ce2144a4049857d3ff9e02de1e",
-    "/app.css": "/app.css?id=bf21b88fc58ce16b4e388d9a112c6c29",
-    "/vendor.js": "/vendor.js?id=1bf42a00db68b9daefba73c9d6dbe50f",
+    "/app.js": "/app.js?id=39a6748854b6a3067197328ad5a1bdbb",
```

Comment:
Not related to this PR, but since this is a vendor Folder,  I wonder if we shouldn't just put it in `.gitignore`?